From a360f974003c5c60c16d019e873314c09d965107 Mon Sep 17 00:00:00 2001
From: Niall Pemberton <niallp@apache.org>
Date: Thu, 30 Sep 2010 00:53:35 +0000
Subject: [PATCH] IO-209 Fix test failing on Unix - freeSpaceWindows() should
 always normalize files to Windows (even for the MockFileSystemUtils on Unix)

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/io/trunk@1002917 13f79535-47bb-0310-9956-ffa450edef68
---
 src/java/org/apache/commons/io/FileSystemUtils.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/org/apache/commons/io/FileSystemUtils.java b/src/java/org/apache/commons/io/FileSystemUtils.java
index c2aa4abf863..3649a631c39 100644
--- a/src/java/org/apache/commons/io/FileSystemUtils.java
+++ b/src/java/org/apache/commons/io/FileSystemUtils.java
@@ -290,7 +290,7 @@ long freeSpaceOS(String path, int os, boolean kb, long timeout) throws IOExcepti
      * @throws IOException if an error occurs
      */
     long freeSpaceWindows(String path, long timeout) throws IOException {
-        path = FilenameUtils.normalize(path);
+        path = FilenameUtils.normalize(path, false);
         if (path.length() > 0 && path.charAt(0) != '"') {
             path = "\"" + path + "\"";
         }
